d3a_api_client/rest_market.py

```diff
@@ -53,7 +53,12 @@ def grid_fees(self, fee_cents_per_kWh):
 
     @logging_decorator('dso_market_stats')
     def list_dso_market_stats(self, selected_markets):
-        transaction_id, posted = self._get_request('dso-market-stats', {"market_slots": selected_markets})
+        # TODO: this is kept for the backward compatibility, should be removed
+        return self.last_market_dso_stats()
```

Comment:
Can you explain a bit further the rationale behind that ? As far as I see, the selected_markets parameter is passed but never used in the method.

Reply:
@spyrostz I left this is in here for backward compatibility , there are still some scripts which use the list_dso_market_stats and they pass market slots, so I did not remove the deprecated attribute to not break raise exceptions, I did add a comment though